pkg/registry/apis/provisioning/jobs/legacy/export_dash.go

```diff
@@ -0,0 +1,241 @@
+package legacy
+
+import (
+	"encoding/json"
+	"fmt"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/grafana/grafana/pkg/infra/db"
+	"github.com/grafana/grafana/pkg/infra/filestorage"
+	"github.com/grafana/grafana/pkg/services/store/kind/dashboard"
+)
+
+func (job *legacyExporter) exportDashboards(helper *commitHelper, keepHistory bool) error {
+	alias := make(map[string]string, 100)
+	ids := make(map[int64]string, 100)
+	folders := make(map[int64]string, 100)
+
+	// // Should root files be at the root or in a subfolder called "general"?
```

Comment:
They should be at the configured folder within the Github repository (e.g. grafana) :)